l10n_us_check_printing_micr/report/print_check.xml

```diff
@@ -0,0 +1,153 @@
+        >
+    <xpath expr="." position="inside">
+        <link
+                    rel="stylesheet"
+                    type="text/scss"
+                    href="/l10n_us_check_printing_micr/static/src/scss/fonts.scss"
+                />
+        <link
+                    rel="stylesheet"
+                    type="text/scss"
+                    href="/l10n_us_check_printing/static/src/scss/report_check_commons.scss"
+                />
+        <link
+                    rel="stylesheet"
+                    type="text/scss"
+                    href="/l10n_us_check_printing_micr/static/src/scss/report_check_bottom.scss"
+                />
+        <link
+                    rel="stylesheet"
+                    type="text/scss"
+                    href="/l10n_us_check_printing_micr/static/src/scss/report_check_middle.scss"
+                />
+        <link
+                    rel="stylesheet"
+                    type="text/scss"
+                    href="/l10n_us_check_printing_micr/static/src/scss/report_check_top.scss"
+                />
+    </xpath>
+</template>
+
+<template id="ckus_check_micr">
+    <div class="ckus_check">
+        <t t-if="page['state'] == 'cancelled'">
+            <div class="o-watermark">VOID</div>
+        </t>
+        <div class="sequence_number" t-esc="page['sequence_number']" />
+        <div class="ckus_date" t-esc="page['date']" />
+        <div class="ckus_amount">
+            <t t-esc="page['amount'].replace(o.currency_id.symbol, '')" />
+        </div>
+        <div class="row ckus_payee_addr_top">
+            <div
+                        class="col-4 text-left"
+                        t-esc="o.company_id.partner_id"
+                        t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
+                    />
+            <div class="col-1" />
+            <div
+                        class="col-5 text-left"
+                        t-esc="o.bank_partner_id"
+                        t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
+                    />
+            <div class="col-2" />
+        </div>
+        <div class="ckus_amount_in_word" t-esc="page['amount_in_word']" />
+        <div class="row ckus_payee_addr">
+            <div
+                        class="col-4 text-left"
+                        t-esc="page['partner_id']"
+                        t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
+                    />
+            <div
+                        class="col-8 float-right"
+                        t-field="o.partner_bank_id.signature_180"
+                        t-options='{"widget": "image", "style":"float: right;"}'
+                    />
+        </div>
+        <div class="ckus_number_micr">
+            <span t-esc="page['sequence_number']" />
+            <span t-esc="o.partner_bank_id.bank_id.routing_number" />
```

Comment:
I commented out the signature line so I can continue testing, however I get the following error when printing the check: